BUILD.gn

```diff
@@ -30,6 +31,9 @@ group("flutter") {
       "//flutter/sky/engine/wtf:wtf_unittests",
       "//flutter/synchronization:synchronization_unittests",
       "//lib/ftl:ftl_unittests",
+      "//lib/txt/examples:txt_example($host_toolchain)",
+      "//lib/txt/tests/txt($host_toolchain)",  # txt_unittests
+      "//lib/txt/tests/unittest($host_toolchain)",  # minikin_unittest
```

Comment:
It would be cleaner if these could be referenced as //lib/txt/examples and //lib/txt/tests, respectively.  That will give us more freedom to add more examples and tests over time without changing both repos.